packages/fxa-auth-server/lib/senders/emails/templates/passwordChangeRequired/index.stories.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { storyWithProps } from '../../storybook-email';
+
+export default {
+  title: 'Emails/passwordChangeRequired',
```

Comment:
If you rebase you'll see all of the others changed to `Templates/`, just a reminder here to do the same to all of these template story files before merging 😁